lib/config_loader.coffee

```diff
@@ -0,0 +1,17 @@
+fs = require 'fs'
+path = require 'path'
+
+
+class ConfigLoader
+
+  constructor: ({@cwd}) ->
+
+
+  load: (done) ->
```

Comment:
I'm not clear why this is a class. This could be a single method that receives `cwd` as a parameter.